101-VM-Virus-Attack-Prevention/README.md

```diff
@@ -0,0 +1,177 @@
+
+5. Open Server Manager and Disable Internet Explorer Enhanced Security Configuration.
+![](images/disable-internet-explorer-enhanced-security-configuration.png)
+
+6. Open Internet Explorer and Download (https://secure.eicar.org/eicar.com.txt)
+
+7. If IE blocks downloading, Go to View Downloads --> Right Click on File and Select Download Unsafe File.
+![](images/download-test-virus-file.png)
+
+8. You will notice that Virus file gets downloads successfully on VM indicating attack was successful.
+
+<a name="mitigate"></a>
+# Mitigate
+
+Azure Security Center gives you recommendations and also allows you to perform remediation steps from the console itself.
+
+1. Go to Home > Security Center - Overview > Compute > Endpoint protection issues > Endpoint Protection not installed on Azure VMs
+
+2. You will notice your VM with name "vm-without-ep" is listed with various other VMs which does not have endpoint protection installed.
+
+3. Select your VM with name "vm-without-ep" and Click on Install on VM. This will install endpoint protection on the VM
+![](images/install-endpoint-protection.png)
+
+# Attack & Detection
+
+Anti-malware Extension (Endpoint Protection) might take 5-10 mins to get installed on a VM. Hence, you will use another VM with name "vm-with-ep" which already has Anti-malware extension installed further for our Demo using loginid and pwd (The VM login username and password is in deployment powershell output)
+
+To perform attack open Internet Explorer and Download (https://secure.eicar.org/eicar.com.txt)
+
+However, this time when you try to download Eicar virus file, it will be immediately detected and will be quarantined by endpoint protection.
+![](images/virus-attack-on-vm-1.png)
+
+## Detection using Endpoint protection client
+
+When you try to access SCEP, you will get restricted access error. 
+![](images/restricted-access-on-endpoint-protection.PNG)
+
+To access endpoint protection client, you need to clean SCEP Policy on Windows workstation or server.
+1. Open **Powershell**
+1. Go to 
+
+   `cd "C:\Program Files\Microsoft Security Client"`
+
+1. then run 
+
+   `.\ConfigSecurityPolicy.exe .\CleanUpPolicy.xml`
+
+1. Now, Go to Start --> Type **System Center Endpoint Protection** --> Go to **History** tab on SCEP Client window.
+1. You will notice Eicar Test Virus file under Quarantined items.
+![](images/scep-history.PNG)
+
+## Detection using centralised OMS.
+
+1. Go to **Azure Portal** --> navigate to resource group **<case no>-virus-attack-on-vm**
+![](images/log-analytics.png)
+
+1. Go to **Log analytics** --> Click on **Log Search** --> Type query `search "eicar"`
+![](images/log-search.png)
+
+   ![](images/search-eicar.png)
+
+You will notice events related to Quarantined items. It might take few minutes for OMS to pull logs from virtual machine, so if you don't get any search results, please try again after sometime.
+
+
+    
+<a name="teardown"></a>
+## Teardown Deployment 
```

Comment:
Script isn't needed for this, removing the resource group is sufficient (via portal)